packages/vite/src/client/overlay.ts

```diff
@@ -171,9 +172,18 @@ export class ErrorOverlay extends HTMLElement {
     this.root.querySelector('.window')!.addEventListener('click', (e) => {
       e.stopPropagation()
     })
+
     this.addEventListener('click', () => {
       this.close()
     })
+
+    this.closeOnEsc = ((e: KeyboardEvent) => {
+      if (e.key === 'Escape' || e.code === 'Escape') {
+        this.close()
+      }
+    }).bind(this)
```

Comment:
Actually if you're using this pattern instead of the one before, we shouldn't need the `.bind(this)` as the closure should use the right one. Would that work? 🤔

Reply:
I have checked that both removing .bind(this) and adding .bind(this) works.

Does removing .bind(this) looks good to you @bluwy ?

Reply:
I didn't see your comment (https://github.com/vitejs/vite/pull/13795#issuecomment-1632104308) when I sent mine:

> I've checked that setting `this.closeOnEsc = (e) => {}` fails on the test that I added.

So does removing it work? :sweat_smile: I'd prefer removing it if possible to keep it cleaner.

Reply:
> So does removing it work? 😅 I'd prefer removing it if possible to keep it cleaner.
Yes! Removing `.bind()` does pass the test that I wrote at ff9395d258b8afaeae2f6230ee05312f33100644. I also tested it on my space as well.

Reply:
> > I've checked that setting `this.closeOnEsc = (e) => {}` fails on the test that I added.
> 
> So does removing it work? 😅 I'd prefer removing it if possible to keep it cleaner.

FYI, what I tried to say when mentioning the failing of the test was that I passes when it should and fails when it shouldn't pass.